From 55ac3d0652e16a7d9c0e0c172f4542f67c0f34d1 Mon Sep 17 00:00:00 2001
From: Ma Wupeng <mawupeng1@huawei.com>
Date: Fri, 11 Nov 2022 09:32:21 +0800
Subject: [PATCH] mm: Drop shmem reliable related log during startup
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 55ac3d0652e16a7d9c0e0c172f4542f67c0f34d1
Modified-by-SEL: No


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4SK3S
CVE: NA

--------------------------------

Message "shmem reliable disabled." will be printed if memory reliable is
disabled. This is not necessary so drop it.

Signed-off-by: Ma Wupeng <mawupeng1@huawei.com>
Reviewed-by: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 mm/mem_reliable.c | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/mm/mem_reliable.c b/mm/mem_reliable.c
index b1bc749532a4..15a84c0f714e 100644
--- a/mm/mem_reliable.c
+++ b/mm/mem_reliable.c
@@ -101,13 +101,8 @@ void mem_reliable_init(bool has_unmirrored_mem, unsigned long *zone_movable_pfn)
 
 void shmem_reliable_init(void)
 {
-	if (!shmem_reliable_is_enabled())
-		return;
-
-	if (!mem_reliable_is_enabled()) {
+	if (!mem_reliable_is_enabled() || !shmem_reliable_is_enabled())
 		shmem_reliable = false;
-		pr_info("shmem reliable disabled.\n");
-	}
 }
 
 void reliable_report_meminfo(struct seq_file *m)
-- 
2.33.0

